src/Components/Publishing/Header/Layouts/Components/FeatureTextHeader.tsx

```diff
@@ -10,11 +10,12 @@ export const FeatureTextHeader: React.SFC<FeatureHeaderProps> = props => {
   const { hero_section } = article
 
   const url = (hero_section && hero_section.url) || ""
+  const isVideo = url.includes("mp4")
   const alt = url.length ? article.title : ""
-  const src = url.length && resize(url, { width: 1200 })
+  const src = url.length && !isVideo && resize(url, { width: 1200 })
 
   return (
-    <TextHeaderContainer>
+    <TextHeaderContainer isEditing={editTitle && true}>
```

Comment:
Minor detail but maybe use `Boolean(editTitle)` instead of `&& true` just for clarity sake